main.go

```diff
@@ -0,0 +1,47 @@
+// tail | (c) 2020 NETWAYS GmbH | GPLv2+
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	readAndPrintFile()
+}
+
+func readAndPrintFile() {
+	buf := bufio.NewReader(os.Stdin)
+	var lineCounter int
```

Comment:
What's `lineCounter` for?

Reply:
to count the lines of the file but if the end of the file is an empty line the program does not count it as a line

Reply:
What about just using `allData`?

Reply:
did not work properly thats why I use lineCounter

Reply:
What exactly "did not work properly"?

Reply:
the len of dataAll is 4 if i have 3 lines + 1 empty line at the end but tail does not count an empty line at the end as a line so tail just ignores it and prints 10 lines upwards from the last-not-empty-line in the file

Reply:
Why don't you just throw the empty line away?